os/booting-on-ecs.md

```diff
@@ -21,21 +21,37 @@ coreos:
      runtime: true
      content: |
        [Unit]
-       Description=Amazon ECS Agent
-       After=docker.service
-       Requires=docker.service
-       Requires=network-online.target
-       After=network-online.target
-       
+       Description=AWS ECS Agent (https://github.com/aws/amazon-ecs-agent)
+       Documentation=https://docs.aws.amazon.com/AmazonECS/latest/developerguide/
+       Requires=network-online.target docker.socket
```

Comment:
My apologies as I may not be understanding systemd correctly, but without Docker and Networking the ecs-agent is totally worthless. The ecs-agent will not be able to register-container-instance, pull docker images, and etc.   Should I change:

```
After=network.target docker.socket
````

to 

```
After=network-online.target docker.socket
```

Again, my apologies, I am still new to systemd. :)